### DISCUSSION TITLE
Max Power Output Setting

### DISCUSSION THREAD

**Discussion by uTZFu** [SonnenladenGmbH/APsystems-EZ1-API-HomeAssistant#39] [Q&A] [answered]
Hi all, first of all: Thanks for this wonderfull integration - makes my life so much easiert to monitor and maintain my mini power plant at home. Second, just a quick question regarding the max power output. The max of 800 Watts - am I right in my assumption that this is independant from another firmware update / release and available right away? Or is there anything else I have to setup  to enable a 600W inverter to deliver 800W? Thanks in advance!

**Comment by mawoka-myblock** [comment, marked as answer]
If you can enable the local API, 800w should be available. To your question: You can set the inverter to any output between 20 and 800w without problem, but make sure it's legal for you ;), as I also have it set to only 600 watts for sure! ;)

**Reply by uTZFu** [reply]
Thanks mate - so I got it right! Since I am not operating the system in Germany I will ramp it up to 800 then :-) thanks for your guidance!

**Reply by mawoka-myblock** [reply]
Sure, no problem!
